libs/output-mapping/src/Table/Result/Metrics.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\OutputMapping\Table\Result;
+
+use Generator;
+
+class Metrics
+{
+    /** @var TableMetrics[] */
+    private array $metrics;
+
+    public function __construct(array $jobResults)
+    {
+        $this->metrics = [];
```

Comment:
deklarace

Reply:
to mi muselo dat hodne prace tohle vymyslet 🤦